src/diffusers/pipelines/__init__.py

```diff
@@ -540,6 +541,7 @@
             AnimateDiffPAGPipeline,
             HunyuanDiTPAGPipeline,
             PixArtSigmaPAGPipeline,
+            StableDiffusion3Pipeline,
```

Comment:
```suggestion
            StableDiffusion3PAGPipeline,
```

to fix the last broken test